configuration/src/configs/database.rs

```diff
@@ -1,14 +1,17 @@
 use crate::configs::deserialize_data_or_env;
+use validator::Validate;
```

Comment:
While you're here, please fix the order of imports:

- std
- 3rd-party
- project's crates
- modules